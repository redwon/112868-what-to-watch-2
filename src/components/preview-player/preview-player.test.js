import React from 'react';
import renderer from 'react-test-renderer';

import PreviewPlayer from './preview-player';

const createNodeMock = (element) => {
  if (element.type === `video`) {
    return {};
  }
  return null;
};

it(`renders correctly`, () => {
  const tree = renderer
    .create(<PreviewPlayer
      src="src"
      poster="poster"
    />, {createNodeMock})
    .toJSON();

  expect(tree).toMatchSnapshot();
});
