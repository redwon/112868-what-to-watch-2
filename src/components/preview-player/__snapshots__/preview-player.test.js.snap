// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<video
  className="player__video"
  muted={false}
  poster="poster"
  src="src"
/>
`;
